placebo/pill.py

```diff
@@ -263,3 +267,9 @@ def _mock_request(self, **kwargs):
         operation = model.name
         LOG.debug('_make_request: %s.%s', service, operation)
         return self.load_response(service, operation)
+
+    def _arn_replacement(self, json_response):
+        """Replacing the recording ARN with a fake ARN value."""
+        json_value = json.dumps(json_response)
+        masked_arn = re.sub(':(\d{12}:)', self.fake_account_number, json_value)
```

Comment:
Just wondering if we could tighten up this regex a little to further limit it to an account number in an ARN.  Probably the leading and trailing ":" would work something that included the "arn:" prefix and intermediate fields might be safer.